### ISSUE TITLE
[bitnami/redis-cluster] Possibility to add labels on created PVC

### ISSUE THREAD

**Issue by ammirator-administrator** [bitnami/charts#25894]
### Name and Version

bitnami/redis-cluster latest

### What is the problem this feature will solve?

Add labels on PVC for all cluster pvc

### What is the feature you are proposing to solve the problem?

Ad the moment to add a label you have to run separate command with kubectl, it would be nice to have this as a helm chart field same as we have in other charts from bitnami, I'm not sure why this one doesn't follow this

### What alternatives have you considered?

just by adding manually, but is not that friendly

**Comment by ammirator-administrator** [comment]
I see this line of code in the volumeClaimTemplates but what it does I'm not sure, and how to pass custom labels there


In the defaultValues in helm chart I do not see a persistence.labels field , Is that even possible with current setup?

```
  volumeClaimTemplates:
    - metadata:
        name: redis-data
        labels: {{- include "common.labels.matchLabels" ( dict "customLabels" .Values.commonLabels "context" $ ) | nindent 10 }}
```

**Comment by ammirator-administrator** [comment]
@javsalgar 
I opened a PR for that can you please review and let me know if is good, should be a easy merge

https://github.com/bitnami/charts/pull/25901

**Comment by carrodher** [comment]
Thank you for opening this issue and submitting the associated Pull Request. Our team will review and provide feedback. Once the PR is merged, the issue will automatically close.

Your contribution is greatly appreciated!

**Comment by ammirator-administrator** [comment]
@carrodher 
Can I get a review on the PR, should be a quick one but I do not see any activity from the bitnami folks at all on it
